simple_salesforce/api.py

```diff
@@ -421,8 +424,40 @@ def _call_salesforce(self, method, url, **kwargs):
         if result.status_code >= 300:
             _exception_handler(result)
 
+        sforce_limit_info = result.headers.get('Sforce-Limit-Info')
+        if sforce_limit_info:
+            self.api_usage = self.parse_api_usage(sforce_limit_info)
+
         return result
 
+    @staticmethod
+    def parse_api_usage(sforce_limit_info):
+        """parse API usage and limits out of the Sforce-Limit-Info header
+
+        Arguments:
+
+        * sforce_limit_info: The value of response header 'Sforce-Limit-Info'
+            Example 1: 'api-usage=18/5000'
+            Example 2: 'api-usage=25/5000;
+                per-app-api-usage=17/250(appName=sample-connected-app)'
+        """
+        result = {
+            'Sforce-Limit-Info': sforce_limit_info
+        }
+
+        api_usage = re.match(r'[^-]?api-usage=(?P<used>\d+)/(?P<tot>\d+)',
+                             sforce_limit_info)
+        pau = r'.+per-app-api-usage=(?P<u>\d+)/(?P<t>\d+)\(appName=(?P<n>.+)\)'
+        per_app_api_usage = re.match(pau, sforce_limit_info)
+
+        if api_usage and api_usage.groups():
+            result['api-usage'] = [int(n) for n in  api_usage.groups()]
```

Comment:
Rather than a bare list of numbers, I think I'd rather see something with a little more context. Maybe `namedtuple`s for both global and per-app usage?